guest_science_projects/sci_cam_image/app/src/main/java/gov/nasa/arc/irg/astrobee/sci_cam_image/CameraController.java

```diff
@@ -402,9 +402,9 @@ public boolean initialize() {
             public void onImageAvailable(ImageReader reader) {
                 final Image image = reader.acquireLatestImage();
                 Date date = new Date();
-                long imageTimestamp = (date.getTime() - SystemClock.uptimeMillis()) + image.getTimestamp();
+                long imageTimestamp = (date.getTime() - SystemClock.uptimeMillis()) + ((long) (image.getTimestamp() * 0.000001));
```

Comment:
I guess it would be slightly cleaner to divide by 1000000 so it uses integer division and no cast is needed. (Not super-important.)

I also think all of this timestamp logic is weird enough that it would make sense to explain in a comment what the inputs mean and why this is the correct formula.